ColorPatterns/Scenes/Game/GameController.swift

```diff
@@ -42,23 +42,27 @@ final class GameController: UIViewController {
     override func viewDidLoad() {
         super.viewDidLoad()
         updateColorBars()
-        setupSwipeDownGesture(direction: .down)
         setupGameTimer()
         setupViewModelDelegate()
         setupScoreLabel()
     }
 
     @IBAction private func colorBarPressed(_ sender: UIButton) {
+        // TODO: Try not to use tags, make an another way to identify buttons
         viewModel.didPressColorBar(colorBar: sender.tag - 1)
         updateColorBars()
         playSound(soundFile: sender.tag - 1)
     }
+    @IBAction private func didSwipeDown(_ sender: UISwipeGestureRecognizer) {
+        updateColorBars()
+    }
 }
 
 private extension GameController {
 
     private func setupGameTimer() {
-        timeLabel.text = gameTimer.timeLeft.formatToString
+        // TODO: You can move timer to viewmodel
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/vandermesis/ColorPatterns/blob/c07cd9a3b7a63e61e86332aeb4fa4cabf8b81c17/ColorPatterns/Scenes/Game/GameController.swift#L64"></span>TODOs should be resolved (You can move timer to viewmode…).<br />
<code>todo</code> <code>GameController.swift:64</code></td>
    </tr>
  </tbody>
</table>